redirects.yaml

```diff
@@ -123,6 +123,22 @@ category/product/landscape/?: https://landscape.canonical.com/
 category/product/(server|ubuntu-server-edition)/?: /server
 category/product/ubuntu-advantage/?: /support
 category/product/ubuntu-light/?: http://lubuntu.net/
+# old certification.ubuntu.com redirects
+certified/desktops/?: "/certified?category=Desktop"
+certified/laptops/?: "/certified?category=Laptop"
```

Comment:
This doesn't exist.

Reply:
Sorry, just realized now, this actually does exist in the new site navigation, so I'll keep it there for now.